src/scripts/background.js

```diff
@@ -266,15 +266,27 @@ const extractTime = (start, end) => {
     }
 }
 
+const cleanIntervals = (intervals = []) => {
```

Comment:
To avoid confusing with intervals and timeout apis you can name this function `extractIntervals`.

Reply:
good point, thank you!